src/features/cli/ReverseShellConnection.jsx

```diff
@@ -0,0 +1,112 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import styled from "@emotion/styled";
+import { ExpandMore } from "@mui/icons-material";
+import {
+  Card, CardContent, Collapse, CardHeader, IconButton, TextField, Button,
+} from "@mui/material";
+import { Box } from "@mui/system";
+import { useCallback } from "react";
+
+const ExpandMoreWrapper = styled((props) => {
+  const {
+    expand, ...other 
+  } = props;
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  return <IconButton {...other} />; // TODO: Review: How to fix?
```

Comment:
Explicitly only pass the props that you actually need for the button.
if you log the content of `other`, you see that it has four properties:

```
aria-label: "settings"
children: {$$typeof: Symbol(react.element), type: {…}, key: null, ref: null, props: {…}, …}
className: "css-1t75bwu"
onClick: () => setExpanded(!expanded)
```

instead of using the spread operator do:

```
const {
  expand,
  children,
  className,
  ...
}
```

and then explicitly pass them to the icon.